### ISSUE TITLE
Document how to debug 

### ISSUE THREAD

**Issue by noerw** [openspace42/aenigma#88]
As the installer always fetches `dna` and `aenigma` on each run of `setup`, it's hard to develop quick fixes. The workaround is to run `setup` with `-dl -pl`, which uses the already fetched versions of the components, thus respects local changes.

I found this after studying the scripts, would be nice to see this documented somewhere.

edit: Also document `-ov` for debug output?

**Comment by nordurljosahvida** [comment]
@noerw correct. `-dl -pl` sources local versions of files even if modified by the user, but in the cloned dir `~/aenigma` and not in the actual installation dir `~/openspace42/aenigma`, and upon installation completion the cloned dir gets removed. Everything has to be documented, but it would be well worth it because this is all part of the underlying frameworks we're developing which would directly benefit other projects as well!
